whylog/assistant/regex_assistant/regex_object.py

```diff
@@ -1,5 +1,7 @@
-from whylog.assistant.regex_assistant.regex import create_obvious_regex
-from whylog.assistant.span_list import SpanList
+from whylog.assistant.const import DataType
+from whylog.assistant.pattern_object import ParamGroup, PatternObject
+from whylog.assistant.regex_assistant.guessing import guess_regex_objects
+from whylog.assistant.regex_assistant.regex import create_obvious_regex, verify_regex
 
 
 class RegexObject(object):
```

Comment:
Why the class is named with 'Object' suffix? Should I think about this class like on the singleton object?